render/id.go

```diff
@@ -0,0 +1,31 @@
+package render
+
+import (
+	"fmt"
+	"strings"
+)
+
+// MakeEndpointID makes an endpoint ID
```

Comment:
Can this (and other) comments be a little more descriptive? It wasn't immediately obvious to me that they're for _rendered_ node IDs, for example.